shortcodes/class-ucf-location-search-form.php

```diff
@@ -67,8 +67,9 @@ public static function callback( $atts, $content = '' ) {
 				$campus    = get_field( 'ucf_location_campus', $post->ID );
 				$thumbnail = get_the_post_thumbnail( $post->ID, 'medium', array( 'class' => 'img-fluid' ) );
 			?>
-			<div class="card<?php echo ( $idx === 0 ) ? '' : ' border-top-0'; ?>">
-				<div class="card-block">
+			<!-- <div class="card<?php // echo ( $idx === 0 ) ? '' : ' border-top-0'; ?>">
+				<div class="card-block"> -->
+				<div class="location-result pt-3">
```

Comment:
Consider changing `pt-3` to `py-3` for even spacing between each location and surrounding dividers--or, change the `my-3` on the `hr`s to `mb-0`

Reply:
Various spacing changes have been made to try to make the spacing even in multiple scenarios.